File: src/main/java/coza/trojanc/receipt/template/process/fields/ProcessedFillLine.java
package coza.trojanc.receipt.template.process.fields;

/**
 * @author Charl Thiem
 */
public class ProcessedFillLine implements ProcessedLineItem {

	private char character = ' ';
	public ProcessedFillLine(){
	}

	public ProcessedFillLine(char character){
		this.character = character;
	}

	public char getCharacter() {
		return character;
	}
}

File: src/main/java/coza/trojanc/receipt/template/process/fields/ProcessedFillLine.kt
package coza.trojanc.receipt.template.process.fields

/**
 * @author Charl Thiem
 */
class ProcessedFillLine : ProcessedLineItem {

    val character = ' '

    constructor() {}

    constructor(character: Char) {
        this.character = character
    }
}
